src/tabs/console/ConsoleTabView.tsx

```diff
@@ -22,23 +23,28 @@ export class ConsoleTabView extends TabView<ConsoleTabState> {
   public render() {
     const controller: ConsoleTabController = this.props.model.getController() as ConsoleTabController;
     return (
-      <RX.View style={[Styles.flex.column, Styles.flex.fill, Styles.justifyFlexEnd]}>
-        <RX.View>
-          {this.state.messages.map((msg, index) => <RX.Text key={index}>[{msg.type}] {this.renderMessage(msg.message)}</RX.Text>)}
+      <RX.ScrollView>
+        <RX.View style={[Styles.flex.column, Styles.flex.fill, Styles.justifyFlexEnd]}>
+          <RX.View>
+            {this.state.messages.map((msg, index) => <RX.Text key={index}>[{msg.type}] {this.renderMessage(msg.message)}</RX.Text>)}
+          </RX.View>
+          <RX.View style={Styles.flex.row}>
```

Comment:
Have you tried keeping the input part (TextInput+Button) outside the ScrollView? I think it could be better from an UX point of view